src/components/AgentDashboard(Rent)/ChangeStatus.js

```diff
@@ -332,6 +337,7 @@ function ChangeStatus() {
                 text="Rented On B8R"
                 onclicked={() => setCloseListingReason("Rented of B8R")}
```

Comment:
the status should read "Rented on B8r" not "Rented of B8R" - can you make the changes in the app?